<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="periodEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setPeriodButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Period</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="teamOneEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="setTeamOneButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Team 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setTeamTwoButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Team 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="teamTwoEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="diconnectButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disonnect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="goalOneEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="goalTwoEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addGoalTwoButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Goal 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addGoalOneButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Goal 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteGoalOneButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Del Goal 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteGoalTwoButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Del Goal 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deletePlayerOneEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="deletePlayerTwoEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePlayerOneButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Del Player 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelDeletionOneButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel Del 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePlayerTwoButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Del Player 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelDeletionTwoButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel Del 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deletePlayerTimeOneEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="deletePlayerTimeTwoEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="setTimeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Time (sec)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>472</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
